spec/models/category_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
- [ ] Please resolve any pending tests. Overall, kindly add more unit tests for your models, controller's methods, private methods, and some integration tests as well haven't been yet written! 👍